android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -501,6 +506,31 @@ private void reLayout(View view) {
         view.measure(MeasureSpec.makeMeasureSpec(getMeasuredWidth(), MeasureSpec.EXACTLY),
                 MeasureSpec.makeMeasureSpec(getMeasuredHeight(), MeasureSpec.EXACTLY));
         view.layout(view.getLeft(), view.getTop(), view.getMeasuredWidth(), view.getMeasuredHeight());
+
+         if(view == playerControlView){
```

Comment:
1/ Can you please move this code in a dedicated function please ?
Something like refreshProgressBarVisibility()

2/ Then move the call to this function in:
2.1/ one call in : initializePlayerControl
2.2/ a second call in: setHideSeekBar

Thanks